next.config.js

```diff
@@ -4,6 +4,7 @@ const redirectsConfig = require('./config/redirects')
 const rewritesConfig = require('./config/rewrites')
 
 // temporary: set all paths as noindex, until we're serving from this project
+// Update the excluded domains to ensure we are indexing content as the io sites get migrated
```

Comment:
I'm going to create a rollout checklist post waypoint and this will be included. I have some notes I'm taking through the process and this has been tracked there.